examples/singleton-types/singleton_types.bal

```diff
@@ -0,0 +1,64 @@
+import ballerina/io;
+
+// The `SwitchStatus` type is a union type of two singleton types
+// defined using string literals, which are simple constant expressions.
+type SwitchStatus "on"|"off";
+
+// The `shouldToggleSwitch` function has two parameters of the `SwitchStatus` type,
+// restricting the arguments to be either "on" or "off". Trying to pass as an argument 
+// a value that cannot be guaranteed to be either "on" or "off"  will result
+// in a compile-time error.
+function shouldToggleSwitch(SwitchStatus currentStatus, SwitchStatus newStatus) 
+        returns boolean {
+    return currentStatus != newStatus;
+}
```

Comment:
Can we do this
```suggestion
function shouldToggleSwitch(SwitchStatus currentStatus, SwitchStatus newStatus) 
        returns boolean =>  currentStatus != newStatus;
```

Reply:
Thought of avoiding expression-bodied functions in case someone got here before checking those examples out. May still be self-explanatory, but to keep things simple we could keep it as is. WDYT?

Reply:
Let's keep it as it is then